src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/TemporaryField.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
+public @interface TemporaryField {
+}
```

Comment:
I'd add  comment in here somewhere to say that this policy refers to @TemporaryField annotations.

Then at least somebody searching on "@TemporaryField" should see this policy.